contributing/documentation/overview.rst

```diff
@@ -35,48 +37,82 @@ Let's imagine that you want to improve the installation chapter of the Symfony
 book. In order to make your changes, follow these steps:
 
 **Step 1.** Go to the official Symfony documentation repository located at
-`github.com/symfony/symfony-docs`_ and `fork the repository`_ to your personal
-account. This is only needed the first time you contribute to Symfony.
+`github.com/symfony/symfony-docs`_ and click on the **Fork** button to `fork the
+repository`_ to your personal account. This is only needed the first time you
+contribute to Symfony.
 
-**Step 2.** **Clone** the forked repository to your local machine (this
-example uses the ``projects/symfony-docs/`` directory to store the documentation;
-change this value accordingly):
+**Step 2.** **Clone** the forked repository to your local machine (this example
+uses the ``projects/symfony-docs/`` directory to store the documentation; change
+this value accordingly):
```

Comment:
OK. I like your idea. In https://github.com/symfony/symfony-docs/pull/6367/commits/4c6b435c36b0213e85d8d015ddf6701fd9777679 commit I've changed some things. Thanks!